libs/sx1302_hal/patches/001-add_compatibility_with_basicstation.patch

```diff
@@ -0,0 +1,27 @@
+From a6ae15dc6709e32f0604b6eeb0f07cc50bde18b1 Mon Sep 17 00:00:00 2001
+From: Marcus Schref <mschref@web.de>
+Date: Mon, 22 Aug 2022 10:06:03 +0200
+Subject: [PATCH] sx1302_hal: add compatibility with basicstation
+
+Add LGW_LBT_ISSUE define used by Basicstation
```

Comment:
Did you try to upstream this? Otherwise, we will need to carry this patch forever.